public/app/plugins/panel/heatmap/types.ts

```diff
@@ -0,0 +1,31 @@
+export {
+  HeatmapColorMode,
+  HeatmapColorScale,
+  HeatmapColorOptions,
+  YAxisConfig,
+  CellValues,
+  FilterValueRange,
+  HeatmapTooltip,
+  HeatmapLegend,
+  ExemplarConfig,
+  RowsHeatmapOptions,
+  PanelOptions,
+  PanelFieldConfig,
+} from './panelcfg.gen';
```

Comment:
```suggestion
export * from './panelcfg.gen';
```
This export can be simplified.